### PR TITLE
User Role Buttons updated

### PR DESCRIPTION
**Pull request by NidhiKJha**
This pull request updates 
- the user list
-Select/Deselect function
Fixes ushahidi/platform#3578

### CODE REVIEW

**Comment by NidhiKJha** [comment]
@Angamanga please review this Pull request :)

File: app/react/common/state/users/users.actions.js [hunk 1]
```diff
@@ -26,6 +27,13 @@ export function requestUsers() {
 
 export function toggleUser(userId) {
     return function action(dispatch) {
-        dispatch({ type: TOGGLE_USER, userId });
+        const id = parseInt(userId, 10);
```
Comment (Angamanga) [inline app/react/common/state/users/users.actions.js]: Converting userId to number for type-consistency.

File: app/react/common/state/users/users.reducers.js [hunk 2]
```diff
@@ -33,9 +34,8 @@ export default function reducer(state = initialState, action) {
                 isLoading: false
             };
         case TOGGLE_USER: {
-            const userId = action.userId.toString();
```
Comment (Angamanga) [inline app/react/common/state/users/users.reducers.js]: No need to convert to string now since it numbers everywhere now, so removed this

**Comment by rowasc** [comment]
@Angamanga do you prefer I close this rather than merge into js-migration? leaving it up to you. :)

**Comment by NidhiKJha** [comment]
@rowasc @Angamanga I'm working on this pull request and I'll submit the code soon. Sorry I missed the previous  comment. Thanks  :)

**Pull request closed without merging**